src/lib/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ check_include_file_cxx(sys/time.h HAVE_SYS_TIME_H)
 check_include_file_cxx(unistd.h HAVE_UNISTD_H)
 check_include_file_cxx(sys/wait.h HAVE_SYS_WAIT_H)
 check_cxx_symbol_exists(mkdtemp "stdlib.h;unistd.h" HAVE_MKDTEMP)
+check_cxx_symbol_exists(mkdtemp "stdlib.h;unistd.h" HAVE_MKSTEMP)
```

Comment:
I guess this supposed to be `check_cxx_symbol_exists(mkstemp...` ?

Reply:
@ni4 Exactly! Thanks for noticing. I we use Windows-specific define, we can leave this out, then?

Reply:
@rrrooommmaaa actually I'm not sure that `mkstemp()`, as part of  POSIX.1, is available on every non-Windows platform, so as for me it is better to use HAVE_MKSTEMP/windows-specific defines separately.

Reply:
Another example - MinGW/MSYS2 will have both mkstemp and Windows defines enabled.